src/main.c

```diff
@@ -806,6 +809,12 @@ static int ubus_poe_info_cb(struct ubus_context *ctx, struct ubus_object *obj,
 			blobmsg_add_string(b, "status", state->ports[i].status);
 		else
 			blobmsg_add_string(b, "status", "unknown");
+		if (state->ports[i].mvolt)
+			blobmsg_add_double(b, "voltage", state->ports[i].mvolt / 1000.0);
+		if (state->ports[i].mampere)
+			blobmsg_add_double(b, "current", state->ports[i].mampere / 1000.0);
```

Comment:
My funny commit about reporting superconductor-like behavior (zero voltage, non-zero current) got lost by github. Short story long, report current and voltage together, under the same predicate:

```
		if (state->ports[i].voltage > 0.0) {
			blobmsg_add_double(b, "voltage", state->ports[i].voltage);
			blobmsg_add_double(b, "current", state->ports[i].current);
		}
```

Reply:
I don't understand this.  If the protocol reports 0 voltage and still a current, then it must mean there is a rounding problem, or the implementation doesn't return the voltage, apparently.
I also noticed (you can calculate it off my example trace) that the watt value is not the same as the calculation of volt * ampere (as it should).

So I believe just returning the values, not trying to interpret/match/understand them is better here, because YMMV.

Reply:
How logical is it to output < 63 mV on a 48V bus, and have a measurable current? That is considered a short circuit in 802.3at/bt terms, and indicated as a fault in the port status. So not, the suggested use case does not make sense.

In a perfect world, one would report V-A if and only if the state is "Delivering power". That's impractical because the status and the measurements arrive to us at different times. That could lead to race conditions such as "delivering power 0.0W". So the next best heuristic is `port->voltage > 0.0`.